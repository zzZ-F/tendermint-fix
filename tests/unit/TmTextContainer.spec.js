import { shallowMount } from "@vue/test-utils"
import htmlBeautify from "html-beautify"
import { TmTextContainer } from "@/index"

describe("TmTextContainer.vue", () => {
  let wrapper
  beforeEach(async () => {
    wrapper = shallowMount(TmTextContainer)
  })

  it("has the expected html structure", () => {
    expect(htmlBeautify(wrapper.html())).toMatchSnapshot()
  })
})
