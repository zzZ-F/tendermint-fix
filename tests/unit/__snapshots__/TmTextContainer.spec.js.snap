// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TmTextContainer.vue has the expected html structure 1`] = `
<div class="tm-text">
  <div class="tm-text-container"></div>
</div>
`;
